api/envoy/data/core/v2alpha/health_check_event.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+
+package envoy.data.core.v2alpha;
+
+import "envoy/api/v2/core/base.proto";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/wrappers.proto";
+
+import "validate/validate.proto";
+import "gogoproto/gogo.proto";
+
+option (gogoproto.equal_all) = true;
+
+// [#protodoc-title: Health check logging events]
+// :ref:`Health check logging <arch_overview_health_check_logging>`.
+
+message HealthCheckEvent {
+  HealthCheckerType health_checker_type = 1;
+  string host_address = 2 [(validate.rules).string.min_bytes = 1];
```

Comment:
@htuch @mrice32 I haven't been fully tracking the convo in https://github.com/envoyproxy/envoy/pull/3478. What are we thinking here for this type of data? Should we be using the full address type?

Reply:
Yes, full address preferable. The only situation it might not be the preferred choice is if we don't have the possibility of there being a port encoded (i.e. not `:80`), it's definitely TCP and it can't be a pipe..

Reply:
One thing from #3478... what happens when we have hosts that appear at multiple levels and priorities? Do we have unique HC events, do we have locality/priority information associated with them?